src/main/java/org/tb/GlobalConstants.java

```diff
@@ -155,7 +155,7 @@
     public static final String OVERTIME_COMPENSATION_TEXT = "Überstundenausgleich";
 
     // pathstrings and iconstrings
-    public static final String ICONPATH = "/tb/images/";
+    public static final String ICONPATH = "/images/";
```

Comment:
Warum wird das geändert?

Reply:
Damit der Context-Path (momentan "/tb") frei gewählt werden kann und ggf. in Zukunft einfach "/" sein kann.